private-templates-service/client/src/store/currentTemplate/actions.ts

```diff
@@ -319,7 +319,7 @@ function initClientSDK(dispatch: any, getState: () => RootState, ): TemplateApi
   const api = new TemplateApi();
   const state = getState();
   if (state.auth.accessToken) {
-    api.setApiKey(0, `Bearer ${state.auth.accessToken!.idToken.rawIdToken}`);
+    api.setApiKey(0, `Bearer ${state.auth.accessToken!.accessToken}`);
```

Comment:
on line 279 there is a call to make `const api = initClientSDK(dispatch, getState);` which is a helper method at the bottom of the file. can you make the setApiKey changes in there so that its consistent for all client SDK related actions?

Reply:
so after going through all the changes in this PR, I feel like we can make this change in that helper method `initClientSDK`, but we can also put it in a util file and call that method in all the actions that require changes in this PR.

Reply:
Done, thanks!